src/components/TotalOrgSummary/TotalOrgSummary.jsx

```diff
@@ -0,0 +1,334 @@
+            className={darkMode ? 'bg-yinmn-blue' : ''}
+          />
+        </Row>
+      </Container>
+    );
+  }
+  return (
+    <Container
+      fluid
+      className={`container-total-org-wrapper py-3 mb-5 ${
+        darkMode ? 'bg-oxford-blue text-light' : 'cbg--white-smoke'
+      }`}
+    >
+      <Row>
+        <Col lg={{ size: 12 }}>
+          <h3 className="mt-3 mb-5">Total Org Summary</h3>
+        </Col>
+      </Row>
+      <hr />
+      <AccordianWrapper title="Volunteer Status">
+        <Row>
+          <Col lg={{ size: 12 }}>
+            <div className="component-container">
+              <VolunteerHoursDistribution />
+            </div>
+          </Col>
+        </Row>
+      </AccordianWrapper>
+      <AccordianWrapper title="Volunteer Activities">
+        <Row>
+          <Col lg={{ size: 12 }}>
+            <div className="component-container">
+              <VolunteerHoursDistribution />
+            </div>
+          </Col>
+        </Row>
+      </AccordianWrapper>
+      <AccordianWrapper title="Global Distribution and Volunteer Status Overview">
+        <Row>
+          <Col lg={{ size: 6 }}>
+            <div className="component-container component-border">
+              <VolunteerHoursDistribution />
+            </div>
+          </Col>
+          <Col lg={{ size: 6 }}>
+            <div className="component-container component-border">
+              <VolunteerHoursDistribution />
+            </div>
+          </Col>
+        </Row>
+      </AccordianWrapper>
+      <AccordianWrapper title="Volunteer Workload and Task Completion Analysis">
+        <Row>
+          <Col lg={{ size: 6 }}>
+            <div className="component-container component-border" style={{ position: 'relative' }}>
+              {(allUserProfiles.fetching || allUsersTimeEntries.loading) && (
+                <div className="d-flex justify-content-center align-items-center0">
+                  <div className="w-100vh ">
+                    <Loading />
+                  </div>
+                </div>
+              )}
+              <div className="d-flex flex-row justify-content-center flex-wrap">
+                {Array.isArray(usersTimeEntries) && usersTimeEntries.length > 0 && (
+                  <>
+                    <VolunteerHoursDistribution
+                      darkMode={darkMode}
+                      usersTimeEntries={usersTimeEntries}
+                      usersOverTimeEntries={usersOverTimeEntries}
+                    />
+                    <div className="d-flex flex-column align-items-center justify-content-center">
+                      <HoursWorkList darkMode={darkMode} usersTimeEntries={usersTimeEntries} />
+                      <NumbersVolunteerWorked usersTimeEntries={usersTimeEntries} />
+                    </div>
+                  </>
+                )}
+              </div>
+            </div>
+          </Col>
+          <Col lg={{ size: 3 }}>
```

Comment:
In dark mode, text is not visible as shown below:

![image](https://github.com/user-attachments/assets/997cb95d-8953-4fe9-aebd-02c5f95a4b5b)

Reply:
This is not part of this PR, it is only Volunteer Hours Distribution

Reply:
I have a question regarding the Volunteer workload display. When the card is collapsed, it appears in dark mode as shown below:
<img width="1489" alt="image" src="https://github.com/user-attachments/assets/e01ed898-d70b-4ee0-b592-3b78cd6a7c98">

However, when expanded, the header of the card appears white while the chart and other elements remain in dark mode, as shown below:
![image](https://github.com/user-attachments/assets/f28570eb-56ac-4dd8-bf92-6304ee939a75)

Is this the expected UI in dark mode? 
Additionally, would it be possible to change the text inside the pie chart ("HOURS WORKED") and the labels to white color in dark mode?

Thank you!